### PR TITLE
[Enhancement] Upgrade CI for PyTorch1.9

### PR DESCRIPTION
**Pull request by Junjun2016**
Will run CI for PyTorch 1.5.0, 1.7.0, 1.8.0, 1.9.0

### CODE REVIEW

**Comment by Junjun2016** [comment]
Hi @xvjiarui 
Finally, the substitution bug has been fixed!
Please review it again!

File: mmseg/.mim/configs [hunk 1]
```diff
@@ -0,0 +1 @@
+../../configs
```
Comment (xvjiarui) [inline mmseg/.mim/configs:1]: Is this file normal?

Comment (Junjun2016) [reply mmseg/.mim/configs:1]: I will remove `mmseg/.mim`.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -21,33 +21,39 @@ jobs:
         run: |
           pip install interrogate
           interrogate -v --ignore-init-method --ignore-module --ignore-nested-functions --exclude mmseg/ops --ignore-regex "__repr__" --fail-under 80 mmseg
-
   build_cpu:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```
Comment (zhouzaida) [inline .github/workflows/build.yml:25]: No need to change the version of ubuntu

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**